<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberGeneric</class>
 <widget class="QWidget" name="NumberGeneric">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>80</height>
   </rect>
  </property>
  <widget class="QLabel" name="unitLearn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLearn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>115</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="genericValueLearn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
